Description: Deactivate CUPS' own log rotating as the system already provides a more sophisticated log rotating

Index: cups-1.4.6/conf/cupsd.conf.in
===================================================================
--- cups-1.4.6.orig/conf/cupsd.conf.in	2011-07-14 13:02:00.199772002 +0200
+++ cups-1.4.6/conf/cupsd.conf.in	2011-07-14 13:02:54.989772934 +0200
@@ -9,6 +9,10 @@
 # for troubleshooting...
 LogLevel @CUPS_LOG_LEVEL@
 
+# Deactivate CUPS' internal logrotating, as we provide a better one, especially
+# LogLevel debug2 gets usable now
+MaxLogSize 0
+
 # Administrator user group...
 SystemGroup @CUPS_SYSTEM_GROUPS@
 @CUPS_SYSTEM_AUTHKEY@
